ms/AuthServices.java

```diff
@@ -40,15 +44,23 @@ public static void main(String args[])
             // Bind this object instance to the name AuthServices in the rmiregistry 
             // Naming.rebind("//" + Configuration.getRemoteHost() + ":1099/CreateServices", obj); 
 
+            //initialize logging
+            log = new Log();
+            fileWriter = log.createLogWriter();
+            //disable logging if the logging initialization failed
+            if(fileWriter == null)
+                logEnable = false;
         } catch (Exception e) {
 
             System.out.println("AuthServices binding err: " + e.getMessage()); 
             e.printStackTrace();
+            if(logEnable)
```

Comment:
Would it be better to put `logEnable` in Log.java instead of here?

Reply:
I feel it makes sense for each microservice to have their own flag of log enable and disable. Decoupling each service's functionality. Open for discussion.